recipe/meta.yaml

```diff
@@ -1,65 +1,58 @@
-{% set version = "2021.5.0" %}
+{% set version = "2021.6.0" %}
 {% set buildnumber = 0 %}
-{% set dal_version = "2021.5.1" %}    # [linux]
-{% set dal_version = "2021.5.0" %}    # [osx or win]
 
 package:
   name: daal4py
   version: {{ version }}
 
 source:
   url: https://github.com/intel/scikit-learn-intelex/archive/{{ version }}.tar.gz
-  sha256: b0298ed384554077e8c49feb8092e895eb0f93dc98bc059cab8aede8a259acd8
+  sha256: a75acdc518f0af262a86952ee1040d5c8b9a9dbd4bf798fd974a917ee5b47e68
   patches:
     - patches/0001-replace-check_estimator_sparse_data-in-tests.patch
-    - winfix_ssize_t.patch  # [win]
-
+    # For patch 0002, see: https://github.com/python/cpython/issues/84006
+    - patches/0002-win-64-py37-distutils.patch  # [win and py==37]
 build:
   number: {{ buildnumber }}
-  # win-32 is not available
-  skip: True                  # [not ((win or linux or osx) and x86_64)]
-  skip: True                  # [py==310]
```

Comment:
We should skip `py<36`, see https://github.com/intel/scikit-learn-intelex/blob/master/daal4py/INSTALL.md#prerequisites

Reply:
Done. Thanks!